src/infra/src/table/short_urls.rs

```diff
@@ -179,11 +171,7 @@ pub async fn list(limit: Option<i64>) -> Result<Vec<ShortUrlRecord>, errors::Err
     if let Some(limit) = limit {
         res = res.limit(limit as u64);
     }
-    let records = res
-        .into_model::<ShortUrlRecord>()
-        .all(client)
-        .await
-        .map_err(|e| Error::DbError(DbError::SeaORMError(e.to_string())))?;
+    let records = res.into_model::<ShortUrlRecord>().all(client).await?;
```

Comment:
This logic moved to [this trait implementation](https://github.com/openobserve/openobserve/pull/5100/files#r1847373471) so it should now happen automatically when `?` converts the error.